test/com/google/javascript/jscomp/LiveVariablesAnalysisTest.java

```diff
@@ -360,11 +360,11 @@ public void testArgumentsArray() {
   @Test
   public void testArgumentsArray_doesNotEscape_destructuredParams() {
     // These cases also cover a crash related to assuming all RESTs have a NAME child.
-    assertNotEscaped("function f([a]) { arguments; }", "a");
-    assertNotEscaped("function f([a] = []) { arguments; }", "a");
-    assertNotEscaped("function f(...[a]) { arguments; }", "a");
-    assertNotEscaped("function f({a}) { arguments; }", "a");
-    assertNotEscaped("function f({a} = {}) { arguments; }", "a");
+    assertNotEscaped("function f([a]) { arguments; }", "a", Wrapper.FUNCTION);
```

Comment:
Should these be `Wrapper.NONE`?

Reply:
Whoops! I managed to re-break the tests. Nice catch; fixed